src/components/motor/server.cpp

```diff
@@ -0,0 +1,232 @@
+#include <components/motor/server.hpp>
+
+#include <common/utils.hpp>
+#include <components/motor/motor.hpp>
+#include <config/resource.hpp>
+#include <rpc/server.hpp>
+
+namespace viam {
+namespace cppsdk {
+
+::grpc::Status MotorServer::SetPower(::grpc::ServerContext* context,
```

Comment:
I'd say staying with `::grpc::Status` is "more right" but in practice the two are likely to be equivalent, since I think there is basically zero chance we would introduce a `grpc` namespace inside the `viam` namespace.